lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -331,20 +331,22 @@ def run_sync(command, stdin = nil)
       stderr = ''
       exit_status = nil
 
+      options = [
+        "-o User=#{@ssh_user}",
+        "-o Port=#{ssh_options[:port]}",
+        "-o IdentityFile=#{ssh_options[:keys][0]}",
```

Comment:
What happens if any of these is unspecified?

Reply:
Is it possible to check values in array or do I have to insert each value separately with `<<` and check that inserted value? For example like this:
```ruby
options << "-o Port=#{port}" if port
options << "-o IdentityFile=#{keys[0]}" if keys
```